.github/workflows/integration-tests.yaml

```diff
@@ -37,7 +28,7 @@ jobs:
       - name: run consumer tests
         run: cd consumer-test && groovy consumerTestController.groovy
         env:
-          REPOSITORY_UNDER_TEST: ${{ steps.repository.outputs.repository }}
+          REPOSITORY_UNDER_TEST: ${{ github.repository }}
           BRANCH_NAME: ${{ steps.branchname.outputs.branchname }}
```

Comment:
Can't you use `${GITHUB_REF#refs/heads/}` directly without the additional `get branch name` step?

Reply:
Should also work. Done.